README.md

```diff
@@ -1 +1,89 @@
-# capstone_one
\ No newline at end of file
+<a name="readme-top"></a>
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+
+- [LIVE]
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [CapstoneProject] <a name="about-project"></a>
+[CapstoneProject] is a capstone project given by Microverse in order to help me practice what I learned in the first module. 
+
+## 🛠 Built With <a name="built-with"></a>
+Vs Code using HTML, CSS and JavaScript.
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- [Feature_1]: Setup a professional Github repository
+- [Feature_2]: Setup microverse linters
+- [Feature_3]:Use images and backgrounds to enhance the look of the website.
+- [Feature_4]: Use JavaScript to manipulate DOM elements
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: to dowload it or clone it to your machine.
+### Setup
+Clone this repository to your desired folder: https://github.com/mariegrace31/capstone_one.git
+### Install
+Install this project with: cd my-project npm install
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+👤 **Author1**
+
+- GitHub: [@mariegrace31](https://github.com/mariegrace31)
+- Twitter: [@mariegracebmg](https://twitter.com/mariegracebmg)
+- LinkedIn: [Marie Grace Bahati](https://linkedin.com/in/marie-gr%C3%A2ce-bahati-546765224)
+<!-- LIVE -->
+Live link: https://mariegrace31.github.io/capstone_one/home.html
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome! And here is the link to the "issues" section:https://github.com/mariegrace31/capstone_one/issues
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, don't hesitate to show your support by liking it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for giving me this opportunity and you too for the time you took going through this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LICENSE -->
```

Comment:
Kindly give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in the acknowledgement section of your README file 😄